app/src/org/commcare/tasks/PurgeStaleArchivedFormsTask.java

```diff
@@ -57,6 +66,9 @@ public static void launchPurgeTask() {
     @Override
     protected Void doInBackground(Void... params) {
         CommCareApp app = CommCareApplication._().getCurrentApp();
+
+        performUnsentAttachmentHotfix(app);
```

Comment:
Wondering what the price is for this procedure. If a device has thousands of saved forms it will take a while for this to run. That said it happens in the background and only happen once, so I guess we are fine.